enterprise/causal-clustering/src/test/java/org/neo4j/causalclustering/catchup/storecopy/CatchupServerIT.java

```diff
@@ -277,10 +277,9 @@ private void listOfDownloadedFilesMatchesServer( NeoStoreDataSource neoStoreData
         assertThat( givenFile, containsInAnyOrder( expectedStoreFiles.toArray( new String[givenFile.size()] ) ) );
     }
 
-    private void assertIndexIdsMatch( PrimitiveLongSet indexIds, NeoStoreDataSource neoStoreDataSource )
+    private void assertIndexIdsAreEmpty( PrimitiveLongSet indexIds )
     {
-        PrimitiveLongSet expectedIndexIds = getExpectedIndexIds( neoStoreDataSource );
-        assertThat( expectedIndexIds, equalTo( indexIds ) );
+        assertTrue( indexIds.isEmpty() );
```

Comment:
`assertThat( indexIds, Matchers.isEmpty() );` would give better output on failure. May as well be inlined too.

Reply:
This is primitive collection so can't do that. But I'll inline and add a custom message

Reply:
You could if you wrote a Matcher :)